php/class-hubspot-contacts.php

```diff
@@ -157,6 +149,20 @@ public static function instance() {
 	public function setup() {
 		global $wp;
 
+		// Adding messages here so we can call __().
+		$messages = array(
+			'error' => __( 'Sorry, an error occurred, please try again.', 'hubspot_subscribe' ),
+			'signed_up' => __( 'You\'re already signed up! Please check your inbox for a confirmation message with a link to change your settings.', 'hubspot_subscribe' ),
```

Comment:
This is technically a violation of WP coding standards. If a string has single quotes in it, you should wrap it with double quotes to avoid escaping.